superset-frontend/src/logger/LogUtils.js

```diff
@@ -19,6 +19,7 @@
 // Log event names ------------------------------------------------------------
 export const LOG_ACTIONS_LOAD_CHART = 'load_chart';
 export const LOG_ACTIONS_RENDER_CHART = 'render_chart';
+export const LOG_ACTIONS_HIDDEN_BROWSER_TAB = 'hidden_browser_tab';
```

Comment:
almost all these event names use a verb to start them, although this event isn't really an action and instead is logging the duration of a state.

Maybe this should be `LOG_DURATION_BROWSER_TAB_HIDDEN = 'duration_browser_tab_hidden'`? I'm not really sure what a good name is here, maybe @john-bodley has thoughts?

Reply:
event is `hidden_browser_tab`, `duration` is just one attribute of the event.

Reply:
I guess my point is, we're not logging an event when the browser tab is hidden, rather we're logging the point when the tab becomes visible again. And the most valuable part of this is the amount of time it spent in the hidden state

Reply:
This is a `timing_event`, like `load_chart` and `render_chart`.
To me `start_offset` is also valuable, I can see when user start to switch browser tab.
Even there is only one valuable attribute, `duration`, this `hidden_browser_tab ` event still need to send other attribute like `event_type` and `trigger_event` so that this event will be grouped with correct dashboard load event.

Reply:
`load_chart` and `render_chart` start with verbs, unlike `hidden_browser_tab`. So I recommend we find a better name for this action. Perhaps `hide_browser_tab`? `make_browser_tab_hidden`?

Reply:
all your recommended names seems not ideal... i chose `hide_browser_tab`.